src/Resources/views/CRUD/Association/edit_many_to_one.html.twig

```diff
@@ -19,7 +19,7 @@ file that was distributed with this source code.
     <div id="field_container_{{ id }}" class="field-container">
         {% if sonata_admin.edit == 'list' %}
             <span id="field_widget_{{ id }}" class="field-short-description">
-                {% if sonata_admin.admin.id(sonata_admin.value) %}
+                {% if sonata_admin.field_description.associationadmin.id(sonata_admin.value) %}
```

Comment:
You can see in the following lines.
```
'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value)
```

The code is similar to the template `edit_one_to_one`.